packages/docs/pages/grid.tsx

```diff
@@ -18,142 +18,205 @@ const ExampleBox: React.FC = ({ children }) => (
 );
 
 const GridPage = () => {
-  const items = [
-    {
-      id: 'examples',
-      title: 'Examples',
-      render: () => (
-        <>
-          <Panel>
-            <Text>The Grid component is useful for creating intrinsically responsive layouts.</Text>
-            <CodePreview scope={{ ExampleBox }}>
-              {/* jsx-to-string:start */}
-              <Grid>
-                <GridItem>
-                  <ExampleBox>Item 1</ExampleBox>
-                </GridItem>
-                <GridItem>
-                  <ExampleBox>Item 2</ExampleBox>
-                </GridItem>
-                <GridItem>
-                  <ExampleBox>Item 3</ExampleBox>
-                </GridItem>
-                <GridItem>
-                  <ExampleBox>Item 4</ExampleBox>
-                </GridItem>
-              </Grid>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Custom layout">
-            <Text>
-              Grid allows you to create custom layouts using combinations of <Code>gridTemplate</Code> and{' '}
-              <Code>gridArea</Code> props.
-            </Text>
-            <CodePreview scope={{ ExampleBox }}>
-              {/* jsx-to-string:start */}
-              {function Example() {
-                const template = `
-              "head head" 80px
-              "nav  main" 200px
-              "foot  foot" 50px
-              / 1fr 5fr;
-              `;
+  return (
+    <>
+      <H1>Grid</H1>
 
-                return (
-                  <Grid gridTemplate={template}>
-                    <GridItem gridArea="head">
-                      <ExampleBox>Header</ExampleBox>
-                    </GridItem>
-                    <GridItem gridArea="nav">
-                      <ExampleBox>Sidebar</ExampleBox>
-                    </GridItem>
-                    <GridItem gridArea="main">
-                      <ExampleBox>Content</ExampleBox>
-                    </GridItem>
-                    <GridItem gridArea="foot">
-                      <ExampleBox>Footer</ExampleBox>
-                    </GridItem>
-                  </Grid>
-                );
-              }}
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Column layout">
-            <Text>
-              You can use the <Code>gridColumns</Code> prop to create columned layouts.
-            </Text>
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          <Code primary>Grid</Code> is a two-dimensional grid-based layout system.
```

Comment:
Let's just say something similar as to what is in the `Flex` PR.
```suggestion
          The <Code primary>Grid</Code> component extends the <Code primary>Box</Code> utility component and provides a <Link href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid" external target="_blank" rel="external nofollow noreferrer">CSS grid</Link> container alongside grid properties.
```